inlong-manager/manager-common/src/main/java/org/apache/inlong/manager/common/pojo/sortstandalone/SortSourceStreamInfo.java

```diff
@@ -17,12 +17,38 @@
 
 package org.apache.inlong.manager.common.pojo.sortstandalone;
 
+import com.google.gson.Gson;
 import lombok.Data;
+import org.apache.pulsar.shade.org.apache.commons.lang.StringUtils;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import java.util.Map;
+import java.util.concurrent.ConcurrentHashMap;
 
 @Data
 public class SortSourceStreamInfo {
     private static final long serialVersionUID = 1L;
+    private static final Logger LOGGER = LoggerFactory.getLogger(SortSourceStreamInfo.class);
     String sortClusterName;
     String sortTaskName;
     String groupId;
+    String extParams;
+    Map<String, String> extParamsMap;
+
+    public Map<String, String> getExtParamsMap() {
+        if (extParamsMap != null) {
+            return extParamsMap;
+        }
+        if (StringUtils.isNotBlank(extParams)) {
+            try {
+                Gson gson = new Gson();
+                extParamsMap = gson.fromJson(extParams, Map.class);
+            } catch (Throwable t) {
+                LOGGER.error(t.getMessage(), t);
```

Comment:
Suggest setting more info in the error log, the exception stack trace has already included the message info.

Reply:
fixed, thx